internal/appconfig/setters.go

```diff
@@ -261,13 +261,13 @@ func (c *Config) SetStatics(statics []Static) {
 	}
 }
 
-func (c *Config) SetVolumes(volumes []Volume) {
+func (c *Config) SetVolumes(volumes []Mount) {
 	c.RawDefinition["mounts"] = volumes
 	// FIXME: "mounts" section is confusing, it is plural but only allows one mount
```

Comment:
Fixme is outdated now, and maybe the function could be `SetMounts`?

Reply:
good idea 👍

Reply:
done db8b4e13